examples/pytorch/nlp/huggingface_models/language-modeling/pruning/eager/run_clm_sparsegpt.py

```diff
@@ -271,8 +270,8 @@ def parse_args():
         help="Transformers parameter: use the external repo")
     
     # Evaluation config
-    parser.add_argument("--tasks", default=["lambada_openai"],
-        help="Usually chosen with ['lambada_openai','hellaswag','winogrande','piqa']",
+    parser.add_argument("--tasks", default="lambada_openai",
+        type=str, help="tasks list for accuracy validation",
```

Comment:
help="tasks list for accuracy validation“，it is not list now.
command usage --tasks "piqa,lambada_openai,hellaswag" with no space.